apps/src/feedback.js

```diff
@@ -267,7 +267,6 @@ FeedbackUtils.prototype.displayFeedback = function (options, requiredBlocks,
       ReactDOM.render(
         <ChallengeDialog
           title={msg.challengeLevelPassTitle()}
-          assetUrl={this.studioApp_.assetUrl}
```

Comment:
Technically a different thing, here we pass `assetUrl` to `ChallengeDialog`, which never uses it (it uses an imported `assetUrl` method instead).